CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ way to update this template, but currently, we follow a pattern:
 
 ## Upcoming version 2020-XX-XX
 
+- [fix] Routes component got double rendered due to Redux container HOC.
+  [#1380](https://github.com/sharetribe/ftw-daily/pull/1380)
```

Comment:
You could mention here that this caused `loadData` to be called twice in client-side page changes.